actors/miner/src/sector_map.rs

```diff
@@ -106,26 +106,18 @@ impl PartitionSectorMap {
         partition_idx: u64,
         sector_numbers: Vec<u64>,
     ) -> anyhow::Result<()> {
-        self.add(partition_idx, BitField::try_from_bits(sector_numbers)?.into())
+        self.add(partition_idx, &BitField::try_from_bits(sector_numbers).unwrap())
```

Comment:
This can panic if passed a sector number that's out of the bitfield range (`u64::MAX`). Let's use `?`.